src/imap.rs

```diff
@@ -1956,7 +1956,7 @@ async fn needs_move_to_mvbox(
         return Ok(false);
     }
 
-    if headers.get_header_value(HeaderDef::ChatVersion).is_some() {
+    if headers.get_header_value(HeaderDef::ChatVersion).is_some() || headers.get_header_value(HeaderDef::ChatDispositionNotificationTo).is_some()  {
```

Comment:
Why do we check this? I guess, a bounce message is what contains `report-type=delivery-status`, no?
Maybe the bug is that `In-Reply-To` isn't handled correctly for bounce messages?

Reply:
I've checked and indeed, this header is not even part of our HeaderDef atm. The problem of whether we actually want to move the NDN still remains. See the comment below.